index.html

```diff
@@ -31,6 +31,148 @@ <h2 class="position">Software Engineer</h2>
           you need coded, don’t hestiate to contact me.
         </p>
       </section>
+
+      <section class="work">
+        <h1>MY RECENT WORK</h1>
+        <hr />
```

Comment:
- [ ] Kindly make sure you have only one `h1` tag in your code because the [SEO](https://developer.mozilla.org/en-US/docs/Glossary/SEO)(Search Engine Optimization) use that to search for your page and having more than one may result in your page not being found🤔. I suggest you use the other `hx` tags `h2-h6` for a more semantic code.✌